common/djangoapps/student/management/commands/bulk_update_email.py

```diff
@@ -0,0 +1,75 @@
+"""
+Management command to bulk update many user's email addresses
+"""
+
+
+import csv
+import logging
+from os import path
+
+from django.core.management.base import BaseCommand, CommandError
+
+from student.models import User
```

Comment:
I am not sure about your particular needs with regards to the User model from student.models, but throughout the platform you'll see:
```
from django.contrib.auth import get_user_model

User = get_user_model()
```

Unless you have a specific reason for using a certain model, I would use this instead